### DISCUSSION TITLE
Allow Test and pre-request scripts to make Env/Collection Variable Updates

### DISCUSSION THREAD

**Discussion by Bullkn0x** [postmanlabs/postman-flows#214] [Ideas]
I'm trying to work with paginated requests and it's not possible with flows right now since variable updates don't seem to work.   Would be a great feature to allow for test and prereq scripts attatched to requests to actually make the env/collection variable updates.

**Comment by PericlesFeltrin** [comment]
Environment variable is not updated in flows :(

You can see more: https://github.com/postmanlabs/postman-flows/discussions/142
